main/acle.md

```diff
@@ -9456,6 +9461,681 @@ are named after. All of the functions have external linkage.
+   svint8_t svld1q_gather[_u64base_s8](svbool_t pg, svint64_t zn, const void *rm);
+   ```
+
+#### LD2Q, LD3Q, LD4Q
+
+Contiguous load two, three or four quadword structures.
+
+ ``` c
+   // Variants are also available for:
+   // _u8, _u16, _s16, _u32, _s32, _u64, _s64
+   // _bf16, _f16, _f32, _f64
+   svint8x2_t svld2q[_s8](svbool_t pg, const int8_t *rn);
+   svint8x2_t svld2q_vnum[_s8](svbool_t pg, const int8_t *rn, uint64_t vnum);
+   svint8x3_t svld3q[_s8](svbool_t pg, const int8_t *rn);
+   svint8x3_t svld3q_vnum[_s8](svbool_t pg, const int8_t *rn, uint64_t vnum);
+   svint8x4_t svld4q[_s8](svbool_t pg, const int8_t *rn);
+   svint8x4_t svld4q_vnum[_s8](svbool_t pg, const int8_t *rn, uint64_t vnum);
+   ```
+
+#### LDNT1B, LDNT1D, LDNT1H, LDNT1W
+
+Contiguous non-temporal load to multi-vector.
+
+ ``` c
+   // All the intrinsics below are [SVE2.1 or SME2]
+   // Variants are also available for _s8
+   svuint8x2_t svldnt1[_u8]_x2(svcount_t png, const uint8_t *rn);
+   svuint8x4_t svldnt1[_u8]_x4(svcount_t png, const uint8_t *rn);
+   svuint8x2_t svldnt1_vnum[_u8]_x2(svcount_t png, const uint8_t *rn,
+                                    int64_t vnum);
+   svuint8x4_t svldnt1_vnum[_u8]_x4(svcount_t png, const uint8_t *rn,
+                                    int64_t vnum);
+
+
+   // Variants are also available for _s16, _f16 and _bf16
+   svuint16x2_t svldnt1[_u16]_x2(svcount_t png, const uint16_t *rn);
+   svuint16x4_t svldnt1[_u16]_x4(svcount_t png, const uint16_t *rn);
+   svuint16x2_t svldnt1_vnum[_u16]_x2(svcount_t png, const uint16_t *rn,
+                                      int64_t vnum);
+   svuint16x4_t svldnt1_vnum[_u16]_x4(svcount_t png, const uint16_t *rn,
+                                      int64_t vnum);
+
+
+   // Variants are also available for _s32 and _f32
+   svuint32x2_t svldnt1[_u32]_x2(svcount_t png, const uint32_t *rn);
+   svuint32x4_t svldnt1[_u32]_x4(svcount_t png, const uint32_t *rn);
+   svuint32x2_t svldnt1_vnum[_u32]_x2(svcount_t png, const uint32_t *rn,
+                                      int64_t vnum);
+   svuint32x4_t svldnt1_vnum[_u32]_x4(svcount_t png, const uint32_t *rn,
+                                      int64_t vnum);
+
+
+   // Variants are also available for _s64 and _f64
+   svuint64x2_t svldnt1[_u64]_x2(svcount_t png, const uint64_t *rn);
+   svuint64x4_t svldnt1[_u64]_x4(svcount_t png, const uint64_t *rn);
+   svuint64x2_t svldnt1_vnum[_u64]_x2(svcount_t png, const uint64_t *rn,
+                                      int64_t vnum);
+   svuint64x4_t svldnt1_vnum[_u64]_x4(svcount_t png, const uint64_t *rn,
+                                      int64_t vnum);
+   ```
+
+#### UMAXQV, SMAXQV, FMAXQV, UMINQV, SMINQV, FMINQV
+
+Max/Min reduction of quadword vector segments.
+
+ ``` c
+   // Variants are also available for:
+   // _s8, _u16, _s16, _u32, _s32, _u64, _s64
+   // _bf16, _f16, _f32, _f64
+   uint8x16_t svmaxqv[_u8]_m(svbool_t pg, svuint8_t zn);
```

Comment:
and here, no _m, _x, _z, because the operation doesn't merge the result into the source vector operand.